cli/tools/coverage/range_tree.rs

```diff
@@ -72,7 +72,7 @@ impl<'rt> RangeTree<'rt> {
   }
 
   pub fn normalize<'a>(
-    rta: &'a RangeTreeArena<'a>,
+    _rta: &'a RangeTreeArena<'a>,
```

Comment:
I don't understand this change. Not a big deal, but do you know why?

Reply:
```rust
error: parameter is only used in recursion
   --> cli/tools/coverage/range_tree.rs:75:5
    |
75  |     rta: &'a RangeTreeArena<'a>,
    |     ^^^ help: if this is intentional, prefix it with an underscore: `_rta`
    |
    = note: `-D clippy::only-used-in-recursion` implied by `-D clippy::all`
note: parameter used here
   --> cli/tools/coverage/range_tree.rs:99:46
    |
99  |           children.push(RangeTree::normalize(rta, head));
    |                                              ^^^
...
113 |         children.push(RangeTree::normalize(rta, head));
    |                                            ^^^
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#only_used_in_recursion
```

Reply:
Happy to remove it in this PR or in a follow-up. Wanted to limit unrelated changes in a toolchain upgrade

Reply:
Dropped it in the latest commit, feel free to squash & merge once green (I don't have merge permissions)